src/processor.rs

```diff
@@ -62,9 +62,9 @@ impl QuizProcessorRef {
     let target_dir = self.src_dir.join("mdbook-quiz");
     fs::create_dir_all(&target_dir)?;
 
-    let mut files = vec!["embed.js", "embed.css"];
+    let mut files = vec!["embed.js", "embed.css", "feedback.js", "feedback.css"];
```

Comment:
Let's make this an opt-in feature. So add a new field to `QuizConfig` that is configured in the `book.toml`, only include the static files if enabled.